tests/runtest.cmd

```diff
@@ -191,6 +192,19 @@ if errorlevel 1 (
     exit /b 1
 )
 
```

Comment:
Yes that was not what I meant. I did not mean to exclude perf tests from normal test runs. I was referring to the fact the script does not exit when someone does runtest.cmd (to only do functional test runs locally). It goes on to also send perf runs to helix.